test/integration/openshift_health_checker/common.go

```diff
@@ -25,9 +27,26 @@ func (p PlaybookTest) Run(t *testing.T) {
 	// A PlaybookTest is intended to be run in parallel with other tests.
 	t.Parallel()
 
-	cmd := exec.Command("ansible-playbook", "-i", "/dev/null", p.Path)
+	// we need to run ansible-playbook with the repository base dir as cwd
+	_, thisFile, _, ok := runtime.Caller(0)
+	if !ok {
+		t.Fatal("could not determine base directory")
+	}
+	repoBase := path.Join(path.Dir(thisFile), "../../..")
+
+	// determine the playbook path relative to repoBase
+	playbook, err := filepath.Abs(p.Path)
+	if err != nil {
```

Comment:
Just an idea: If you reuse this pattern (which is super common in Go) it may be worth functionizing it:

```golang
func failOnError(err Error):
    if err != nil {
        t.Fatal(err)
    }
```